pkgs/top-level/all-packages.nix

```diff
@@ -7910,6 +7910,11 @@ with pkgs;
   tcl-8_5 = callPackage ../development/interpreters/tcl/8.5.nix { };
   tcl-8_6 = callPackage ../development/interpreters/tcl/8.6.nix { };
 
+  # TODO(ekleog): remove after 19.03 branch-off
+  # TODO(ekleog): add ‘wasm’ alias to ‘ocamlPackages.wasm’ after 19.09
+  # branch-off
+  wasm = lib.warn "‘wasm’ package has been renamed ‘proglodyte-wasm’"
+                  proglodyte-wasm;
```

Comment:
We have `pkgs/top-level/aliases.nix` for that.

Reply:
Didn't know about it, thanks! :) I've just pushed a version using `aliases.nix`, though the `lib.warn` appears to still be necessary as I didn't get any warning when trying to build without it.

Reply:
Didn't know about it, thanks! :) I've just pushed a version using aliases.nix, though the lib.warn appears to still be necessary as I didn't get any warning when trying to build without it.